gstaichi/runtime/cuda/jit_cuda.cpp

```diff
@@ -345,29 +372,24 @@ std::string JITSessionCUDA::compile_module_to_ptx(
 
     Note there's an update for "separate-const-offset-gep" in llvm-12.
   */
-  module_pass_manager.add(llvm::createLoopStrengthReducePass());
-  module_pass_manager.add(llvm::createIndVarSimplifyPass());
-  module_pass_manager.add(llvm::createSeparateConstOffsetFromGEPPass(false));
-  module_pass_manager.add(llvm::createEarlyCSEPass(true));
+  legacy_pm.add(llvm::createLoopStrengthReducePass());
+  // legacy_pm.add(llvm::createIndVarSimplifyPass());
+  legacy_pm.add(llvm::createSeparateConstOffsetFromGEPPass(false));
+  legacy_pm.add(llvm::createEarlyCSEPass(true));
+  // module_pass_manager.add(llvm::createLoopStrengthReducePass());
+  // module_pass_manager.add(llvm::createIndVarSimplifyPass());
+  // module_pass_manager.add(llvm::createSeparateConstOffsetFromGEPPass(false));
+  // module_pass_manager.add(llvm::createEarlyCSEPass(true));
```

Comment:
I don't like commented stuffs. Why are they commented?

Reply:
(well, i was wiating for benchmark results first; will remove now)

Reply:
the code is stored in git anyway.

Reply:
It's basically the old legacy passes, that we deleted.

Reply:
(*migrated to new passes)

Reply:
this by the way, is why I asked for us to have benchmarks on CI in the first place :) Precisely for LLVM pass manager upgrade :)